.github/workflows/sync-tflm.yml

```diff
@@ -31,12 +39,14 @@ jobs:
 
       - name: Sync the code, commit and push
         run: |
+          source environment
+          python3 -m pip install -r conf/requirements.txt
```

Comment:
@tcal-x I understand this might be a tiny bit controversial, but my reasoning behind this is: this is used only to be able to run a script that's contained within the CI. It's not required in regular builds, it does not affect users in any way.

We could use conda if it was used globally across the project, otherwise it's just making things complex for no real benefit.